staging/src/k8s.io/component-base/logs/example/cmd/logger.go

```diff
@@ -51,6 +51,7 @@ func NewLoggerCommand() *cobra.Command {
 		},
 	}
 	o.AddFlags(cmd.Flags())
+	logs.AddFlags(cmd.Flags())
```

Comment:
Should `o.AddFlags` not be enough? why call both `o.AddFlags` and `logs.AddFlags`?

Reply:
I had the same thought and even tried it, but it broke kubelet:
- `o.AddFlags` (or rather, `BindLoggingFlags` but conceptually they are the same) is expected to add flags just for the fields in `LoggingConfiguration`. Those flags then get the "DEPRECATED, use config file" marker in kubelet.
- `logs.AddFlags` is for global flags that don't have an entry in the config (klog, "log-flush-frequency").

Reply:
/hold

This PR is incomplete without that other commit. Probably it is better to merge this together in a different, larger PR.

Reply:
https://github.com/kubernetes/kubernetes/pull/105042 supersedes this PR.